src/test/ui/macros/meta-variable-depth-outside-repeat.rs

```diff
@@ -0,0 +1,12 @@
+#![feature(macro_metavar_expr)]
+
+macro_rules! metavar {
+    ( $i:expr ) => {
+        ${length(0)}
+        //~^ meta-variable expression `length` with depth parameter must be called inside of a macro repetition
```

Comment:
```suggestion
        //~^ ERROR meta-variable expression `length` with depth parameter must be called inside of a macro repetition
```
(We should remove support for "unelaborated" annotations like this from compiletest.)